index.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- let's load DS styles first, do not move this line down -->
+    <link rel="stylesheet" href="node_modules/design-system/dist/css/styles.css">
```

Comment:
what's the reason for moving it over the title? does it behave differently in such case?

Reply:
when it was at the bottom DS styles were overriding some post-editor styles